### PR TITLE
Add npm to Lando tooling (fixes #40).

### LINKED ISSUE #40: Lando: Add frontend tools to local dev. environment
Add `lando npm` and `lando yarn` commands and run `yarn` or `npm install` on build step by default. 

Apply this also on the https://github.com/wunderio/WunderTools

### PR DESCRIPTION
**Pull request by tormi**
## Features

- Adds `node` service with [Lando default version](https://docs.devwithlando.io/tutorials/node.html#supported-versions) (currently 10). 
- Adds `lando npm` tool.
- Adds automatic `npm install` at build time.

### CODE REVIEW

File: .lando.yml [hunk 1]
```diff
@@ -15,9 +15,17 @@ tooling:
     cmd:
       - ./vendor/bin/codecept
       - "--env=lando"
+  npm:
```
Comment (floretan) [inline .lando.yml]: We should only use one tool by default, we have too many projects with a mixture and that's causing inconsistencies. CircleCI is configured to use yarn, so it would make sense to use that.

File: .lando.yml [hunk 2]
```diff
@@ -15,9 +15,17 @@ tooling:
...
+    description: Runs npm commands
+    service: node
+  yarn:
+    description: Runs Yarn commands
+    service: node
 
 services:
   appserver:
+    build:
```
Comment (floretan) [inline .lando.yml]: This is not related to the frontend tooling.

File: .lando.yml [hunk 3]
```diff
@@ -35,6 +43,12 @@ services:
     type: mailhog
     hogfrom:
       - appserver
+  node:
+    type: node:10
+    build:
+      # Uncomment preferable tool for installing dependencies
```
Comment (floretan) [inline .lando.yml]: Here too, we should only have one tool by default.

Also, since we provide a package.json in the repository, we don't need to comment this out. This will be different for existing projects, but we should remember that this repository is just a starting point.

File: README.md [hunk 4]
```diff
@@ -21,3 +21,15 @@ Have a look at the file for details, but in short this is how it works:
 - Create a custom docker image for Drupal and nginx, and push those to a docker registry (typically that of your cloud provider).
 - Install or update our helm chart while passing our custom images as parameters.
 - The helm chart executes the usual drush deployment commands.
+
+## Local development
+
+Quick start: install [Lando](https://docs.devwithlando.io/), add your project's name to `.lando.yml` and run `lando start`.
+
+### Useful commands
```
Comment (floretan) [inline README.md]: Let's not duplicate other tools' documentation. This is bound to become outdated, and people should get used to using `--help` with any tools.

Comment (tormi) [reply README.md]: Removed.

File: .lando.yml [hunk 5]
```diff
@@ -35,6 +38,10 @@ services:
     type: mailhog
     hogfrom:
       - appserver
+  node:
```
Comment (floretan) [inline .lando.yml]: I think we generally don't have a need to run node directly, so I wouldn't add it by default.

Comment (tormi) [reply .lando.yml]: `npm` tool uses `node` service.

Comment (floretan) [reply .lando.yml]: Ah, yes, of course that makes sense. I read too quickly and didn’t realize this is a service and not a command.
